llama_hub/earning_calls_transcripts/README.md

```diff
@@ -0,0 +1,86 @@
+# EARNING CALL TRANSCRIPTS LOADER
+
+This loader fetches the earning call transcripts of US based companies from the website [discountingcashflows.com](https://discountingcashflows.com/). It is not available for commercial purposes
+
+Install the required dependencies
+
+```
+pip install -r requirements.txt
+```
+
+The Earning call transcripts takes in three arguments
+
+* Year
+* Ticker symbol
+* Quarter name from the list ["Q1","Q2","Q3","Q4"]
+
+## Usage
+
+```python
+from llama_index import download_loader
+
+IMDBReviewsloader = download_loader('EarningCallTranscripts')
+
+loader = EarningCallTranscripts(2023,'AAPL','Q3')
+docs = loader.load_data()
+```
+
+The metadata of the transcripts are the following
+
+* ticker
+* quarter
+* date_time
+* speakers_list
+
+## Examples
+
+#### Llama Index
+```python
+from llama_index import download_loader
+from llama_index import GPTVectorStoreIndex, download_loader
```

Comment:
GPTVectorStoreIndex is outdated, it's now just VectorStoreIndex

Reply:
Yes I have done this change, thanks